src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -234,6 +246,13 @@ g.test('onscreenCanvas,snapshot')
 
     TODO: Snapshot canvas to jpeg, webp and other mime type and
           different quality. Maybe we should test them in reftest.
+
+    check 1:
+      canvas(colorSpace)->img(colorSpace)->canvas(colorSpace).drawImage->canvas(colorSpace).getImageData->actual
+      hard coded data->expected
+    check 2:
+      canvas(colorSpace)->img(colorSpace)->canvas(diffColorSpace).drawImage->canvas(diffColorSpace).getImageData->actual
+      hard coded data->ImageData(colorSpace)->canvas(diffColorSpace).putImageData->canvas(diffColorSpace).getImageData->expected
```

Comment:
These comments are describing behavior of the `checkImageResult` function right? Maybe it's better to put it at the helper function as it's called at >1 places. (called at 'offscreenCanvas,snapshot' as well)

Reply:
done